cppcache/integration/framework/Cluster.cpp

```diff
@@ -161,7 +163,7 @@ Server::Server(Cluster &cluster, std::vector<Locator> &locators,
     hostname = "ip6-localhost";
   }
 
-  auto port = static_cast<uint16_t>(0);
+  auto port = Framework::getAvailablePort();
```

Comment:
Geode server will select its own ephemeral port if we send 0. We should preserve this default behavior in the tests. If you test needs to know the specific port of the server then that test should use this method to get a free port and assign it to the server in the cluster config.